localstack/services/generic_proxy.py

```diff
@@ -965,8 +966,8 @@ def handler(request, data):
         method = request.method
         headers = request.headers
         headers[HEADER_LOCALSTACK_REQUEST_URL] = str(request.url)
-
-        response = modify_and_forward(
+        headers[HEADER_LOCALSTACK_AUTHORIZATION] = headers.get("Authorization", "")
```

Comment:
This is here the magic happens :) We copy the user authorization header and store it in a custom header, that will be plucked in the backend integration